index.php

```diff
@@ -9,34 +9,80 @@
   <!-- main posts loop -->
   <section id="posts" class="container">
 <?php
-if( have_posts() ) {
-  while( have_posts() ) {
-    the_post();
+  $categories = get_terms('category', array('orderby' => 'id', 'hide_empty' => false));
+
+  if ($categories) {
+    $i = 1;
+    foreach($categories as $category) {
```

Comment:
what u think it would do. `$i` is the index of the iteraiton and `$category` is w/e is in the actual array position…

in this case `$i` starts from 1 instead of the normal 0… i guess it makes sense, i just think its more readable to have the index (`$i`) inside the foreach so u know it increments on each iteration…